LinkedIdeas/CanvasViewController.swift

```diff
@@ -219,15 +305,31 @@ extension CanvasViewController {
   }
   
   override func mouseDragged(with event: NSEvent) {
+    if (dragCount <= 3) {
+      Swift.print("[mouseDragged]")
+      dragCount += 1
+    }
+    
     let point = convertToCanvasCoordinates(point: event.locationInWindow)
     
     // Decision: which actions to trigger
     // given the context and events that happen
     
     switch currentState {
     case .selectedElement(let element):
-      guard let concept = element as? Concept else { return }
-      drag(concept: concept, toPoint: point)
+      if (event.modifierFlags.contains(.shift) || didShiftDragStart) {
```

Comment:
Maybe you could extra this statement into a variable with a descriptive name about what is being checked here.